pkg/antctl/raw/set/command.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package set
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	flowaggregatorconfig "antrea.io/antrea/pkg/config/flowaggregator"
+)
+
+// Command is the support bundle command implementation.
+var Command *cobra.Command
+
+type FlowAggregatorConfigMutator func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error
+
+var mutator map[string]FlowAggregatorConfigMutator
+
+var example = strings.Trim(`
+  Enable clickhouse
+  $ antctl set clickhouse.enable=true
+  Update database
+  $ antctl set clickhouse.database=name
+  Update databaseURL
+  $ antctl set clickhouse.databaseURL=Http://xxxxx
+  Update debug
+  $ antctl set clickhouse.debug=true
+  Update compress
+  $ antctl set clickhouse.compress=true
+  Update commitInterval
+  $ antctl set clickhouse.commitInterval=10s
+  Update ttl
+  $ antctl set clickhouse.ttl=100
+  Update externalflowcollectoraddr
+  $ antctl set flowcollector.externalflowcollectoraddr=<IP>:<port>[:<proto>]
+  Enable flow-collector
+  $ antctl set flowcollector.enable=true
+`, "\n")
+
+func init() {
+	Command = &cobra.Command{
+		Use:   "set",
+		Short: "Update clickhouse/flow-collector parameters used in flow aggregator",
+	}
+	mutator = map[string]FlowAggregatorConfigMutator{
+		"clickhouse.enable": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.ClickHouse.Enable, value)
+		},
+		"clickhouse.database": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setString(&c.ClickHouse.Database, value)
+		},
+		"clickhouse.databaseURL": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setString(&c.ClickHouse.DatabaseURL, value)
+		},
+		"clickhouse.debug": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.ClickHouse.Debug, value)
+		},
+		"clickhouse.compress": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolPOrFail(&c.ClickHouse.Compress, value)
```

Comment:
I think you could use:
```go
c.ClickHouse.Compress = new(bool)
return setBoolPOrFail(c.ClickHouse.Compress, value)
```
to avoid defining a new function?

Reply:
Make sense. I've modified it. Thank you.